user-guide/Standard_Apps/EPM/EPM_I-DOCSIS/I-DOCSIS_Create_CCAP_CM_pair.md

```diff
@@ -87,23 +99,29 @@ The elements will be created with the following specifications:
 
 1. In the lower right corner, click *Execute*.
 
-   This will open a wizard with several steps.
+   This will open a pop-up window.
+
+1. Click *Create bulk*.
 
-1. Below the **Run automation** script text, enter the path of the CSV file.
+1. Below the **Create Bulk** script text, enter the path of the CSV file.
 
    > [!NOTE]
-   > The CSV file has to have the following structure: ElementName, Be_DMA, Collector_DMA, Protocol, IpAddress, CommunityString, Network, Market, Hub, SystemUser, SystemPass.
+   >
+   > - The CSV file must be present on the server running the DMA.
+   > - The CSV file has to have the following structure: ElementName, Ccap_DMA, Collector_DMA, Protocol, IpAddress, GetCommunityString, SetCommunityString,GetCommunityStringColl, SetCommunityStringColl, Network, Market, Hub, SystemUser, SystemPass.
 
-1. Click *Run automation*.
+1. Click *Create Bulk*.
 
 The elements will be created with the specifications in the CSV file of each row.
 
 Here is an example of the CSV file with the mandatory headers and the content of each row.
 
-| ElementName | Be_DMA               | Collector_DMA        | Protocol                  | IpAddress    | CommunityString | Network | Market          | Hub           | SystemUser | SystemPass |
-|-------------|----------------------|----------------------|---------------------------|--------------|-----------------|---------|-----------------|---------------|------------|------------|
-| filename1   | EPM-BE-PUE-LAB-DMA03 | EPM-BE-PUE-LAB-DMA03 | CISCO CMTS CCAP Platform  | 127.0.0.100  | AnyText          | BAJIO   | AGUASCALIENTES | CTC-AGS       | us1        | Pass1      |
-| filename2   | EPM-BE-CLN-LAB-DMA04 | EPM-BE-PUE-LAB-DMA03 | Arris E6000 CCAP Platform | 127.0.0.101  | AnyText          | BAJIO   | AGUASCALIENTES | CTC-AGS       | us2        | Pass2      |
+| ElementName | Ccap_DMA    | Collector_DMA | Protocol                  | IpAddress    | GetCommunityString | SetCommunityString | GetCommunityStringColl | SetCommunityStringColl | Network         | Market          | Hub            | SystemUser | SystemPass |
+|-------------|-------------|----------------|---------------------------|--------------|---------------------|---------------------|------------------------|------------------------|-----------------|------------------|-----------------|-------------|------------|
+| filename1   | SLC-H62-G05 | SLC-H62-G05    | CISCO CBR-8 CCAP Platform | 127.0.0.100  | getPublic           | setprivate          | collectorget           | collectorset           | GLOBAL NETWORK  | EAST MARKET 01   | EAST HUB 01     | US1         | 123        |
+| filename2   | SLC-H62-G05 | SLC-H62-G05    | CISCO CBR-8 CCAP Platform | 127.0.0.101  | getprivate          | setPublic           | collectorget           | collectorset           | GLOBAL NETWORK  | EAST MARKET 01   | EAST HUB 01     | US2         | 123        |
 
```

Comment:
Please remove names related to the internal environments. For example SLC-H62-G05 is the name of our internal server. Customers don't need to know that.
On top of that, please use a different IP address for the examples, maybe: 10.11.12.11 and 10.11.12.12
127 subnet is reserved for loopback, see https://serverfault.com/questions/402938/what-is-the-rest-of-the-127-0-0-0-8-address-space-used-for

Reply:
I've replaced the DMA names with more generic, fake DMA names. OK like this?

Reply:
Hi @MariekeGO , yes it is ok